crates/turborepo-lib/src/opts.rs

```diff
@@ -0,0 +1,154 @@
+use anyhow::{anyhow, Result};
+use serde::{Deserialize, Serialize};
+
+use crate::{
+    cli::{Command, DryRunMode, EnvMode, LogPrefix, RunArgs},
+    daemon::{DaemonClient, DaemonConnector},
+    Args,
+};
+
+#[derive(Debug)]
+pub struct Opts<'a> {
```

Comment:
Do we care about the opts having a lifetime? Would it make it easier if they owned their data? It's not a ton of data...

Reply:
Right now it can go either way. I think in the future I'd like a single `'run` lifetime that's tied to either `CommandBase` or a `Run` struct. And that can own stuff like Args that then in turn get transformed to `Opts`. For now I don't think we have to make that call until this is used.